.ci/unit_test

```diff
@@ -52,13 +52,13 @@ TEST_PACKAGES="${TEST_PACKAGES:-"cmd pkg"}"
 
 RUN_NEGATIVE="${RUN_NEGATIVE:-"true"}"
 
-GINKGO_COMMON_FLAGS=("-r" "-timeout=1h0m0s" "--progress" "-slowSpecThreshold=180" "-mod=vendor")
```

Comment:
Why remove these two flags?
I understand why you'd like to remove `slowSpecThreshold`, but won't `progress` be beneficial since we have so many tests here?

Reply:
But I have added `--show-node-events` instead of `progress` and `--fail-on-pending` to prevent skipping the test cases